spec/actors/hydranorth/generic_files/actor_spec.rb

```diff
@@ -1,12 +1,10 @@
 require 'spec_helper'
 
 describe Hydranorth::GenericFile::Actor do
-  include ActionDispatch::TestProcess
-  
-  let(:user) { FactoryGirl.find_or_create(:user) }
-  let(:generic_file) { FactoryGirl.create(:generic_file) }
+  let(:user) { FactoryGirl.build(:user) }
+  let(:generic_file) { FactoryGirl.build(:generic_file) }
```

Comment:
Here's a great example of showcasing the bottlenecks of Fedora. In this particular test, we do not need a creation of a GF, building the GF in memory will do just fine. This test went from taking 10 seconds to < 1 second simply because of this small change. Lesson? Try not to persist to the Fedora if we do not need to.